lib/metamodelutil.js

```diff
@@ -89,7 +89,7 @@ function createNameTable(priorModels, metaModel) {
                 if (!findDeclaration(modelFile, type)) {
                     throw new Error(`Declaration ${type} in namespace ${namespace} not found`);
                 }
-                table[localName] = {namespace, name: type};
+                table[localName] = localName !== type ? {namespace, name: localName, resolvedName: type} : {namespace, name: type};
```

Comment:
So here, if the imported type is aliased, you are replacing the name of the type with its local name.

Which will then be used to assign the super type of the concept here https://github.com/accordproject/concerto/blob/85366be18b5deb3efaa6acfbc7319b14a0219c05/packages/concerto-core/lib/introspect/classdeclaration.js#L70

Which would then fail to resolve, because the runtime will try to resolve the type using that name here https://github.com/accordproject/concerto/blob/85366be18b5deb3efaa6acfbc7319b14a0219c05/packages/concerto-core/lib/introspect/classdeclaration.js#L163

because it won't exist.